ChangeLog

```diff
@@ -9,6 +9,11 @@ Release date: TBA
 ..
   Put new features here and also in 'doc/whatsnew/2.13.rst'
 
+* ``used-before-assignment`` now considers that assignments in the try block
```

Comment:
Can you add a comment about this to the description of the message? I have been working on auto generating documentation for each individual message and it would be helpful if the documentation displayed this.